### PR TITLE
fast, dirty base for documentation

### PR DESCRIPTION
**Pull request by gregglind**
- remove examples
- move https://github.com/gregglind/template-shield-study/pull/18 to here.
- modify README
- move howToShieldStudy into docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,9 +68,26 @@ Using this, you get this analysis FOR FREE (and it's fast!)
...
+- open an issue
+- hack and file a PR
+
+
+## history of major versions
+
+- v4.x: (proposed)  additional functions for common cases
+- v4: First `.jsm` release.  Uses packet format for PACKET version 3.
+- v3: Attempt to formalize on `shield-study` PACKET version 3.  Jetpack based.  Prototype used for `raymak/page-reload`.  All work abandoned, and no formal npm release in this series.  Work done at `v3-shield-packet-format` branch.  LAST JETPACK (addon-sdk) RELEASE.
+s v2: Code refactor to es6 `class` with event models.  Added cli tooling.  Packet format is still arbitrary and per-study.  Jetpack based.  Last used in studies in Q2 2017.
```
Comment (pdehaan) [inline README.md]: Nit: this should be a hyphen or something to preserve bulleting.

File: docs/survival.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+> Look
+
+DARK CLOUDS block the sun.
+
+On the beach is a HANDBOOK here.
+
+There are TELEMETRY PROBES here.
+
+There is a DEVELOPMENT ENVIROMENT here.
```
Comment (pdehaan) [inline docs/survival.md]: typo: ENVIRONMENT

File: docs/survival.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+## Do I
+
+
+
+## But I like making User Interface?
+
+Don't we all?!  Mocks are fun!  Styling is fun!
+
+starting with Telemetry makes soe of the... unexpected decisions make sense.
```
Comment (pdehaan) [inline docs/survival.md]: typo: some

File: docs/survival.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+## Wait, did you say legacy addon?
+
+Yes.  Web Extensions CAN'T SEND TELEMETRY.  We need Firefox (chrome) privileges to access the `TelemetryController.jsm`.  That meanss
```
Comment (pdehaan) [inline docs/survival.md]: typo: "means"

File: docs/survival.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+`manifest.json`
+
+Getting the probes to firefox
+
+
+
+"At least both branches are equally bad":  A plea for experimental controls
+
+
+FInding shelter
```
Comment (pdehaan) [inline docs/survival.md]: typo: "Finding"

File: docs/survival.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+As you look up, a light rain starts to fall.  You wonder how you will make a fire to keep warm, as your clothes start to soak.  The first chills of hypothermia ripple your flesh.
+
+Legacy addon development has challenges.
+
+
+> examine handbook.
+
+(taking handbook)
+
+SURVIVAL IN HARD ENVIROMENTS
```
Comment (pdehaan) [inline docs/survival.md]: typo: ENVIRONMENTS

**Comment by biancadanforth** [comment]
Closing since this is incorporated into #113 .

**Pull request closed without merging**